backend/src/components/user.js

```diff
@@ -13,29 +13,37 @@ async function getEdxUserByDigitalId(accessToken, digitalID, correlationID) {
     const params = {
       params: {
         digitalId: digitalID,
-      }
+      },
     };
-    return await getDataWithParams(accessToken, config.get('edx:edxUsersURL'), params, correlationID);
+    return await getDataWithParams(
+      accessToken,
+      config.get('edx:edxUsersURL'),
```

Comment:
can we change config name to iosas?

Reply:
Yes - However I just formatted this file, so Im not certain what this is in reference to. I think this entire function will be deleted - Can we address it then?